accessibility_monitoring_platform/apps/common/templates/common/phase_banner.html

```diff
@@ -6,7 +6,7 @@
             {% if django_settings.AMP_PROTOTYPE_NAME and django_settings.AMP_PROTOTYPE_NAME != 'TEST' %}
                 report
             {% else %}
-                <a href="{% url 'common:issue-report' %}?page_url={{ request.path }}"
+                <a href="{% url 'common:issue-report' %}{% if user.is_authenticated %}?page_url={{ request.path }}{% endif %}"
```

Comment:
I think it might be better to hide the entire link and only show it if the user is authenticated

Reply:
Okay. 4de3be67ec303c8f10eaed377405c4412dc40391